hex/FormTest2.resx

```diff
@@ -120,6 +120,15 @@
   <metadata name="statusStrip1.TrayLocation" type="System.Drawing.Point, System.Drawing, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a">
     <value>17, 17</value>
   </metadata>
+  <data name="hex21.QueryHitAddresses" mimetype="application/x-microsoft.net.object.binary.base64">
```

Comment:
QueryHitAddresses is the list of addresses which match the search criteria when you search for something. Without this code, the list will always return null.